examples/fluids/README.md

```diff
@@ -72,14 +72,26 @@ The following options are common among all problem types:
   - Frequency of output, in number of steps. `0` has no output, `-1` outputs final state only
   - `10`
 
-* - `-continue`
-  - Continue from previous solution
-  - `0`
-
 * - `-output_dir`
   - Output directory
   - `.`
 
+* - `-output_add_stepnum2bin`
+  - Whether to add step numbers to output binary files
+  - `false`
```

Comment:
Thanks for making this PR. Could we simplify life by writing the step number into `ns-time.bin` so that it can just be read, instead of placing semantic meaning on the file name?

Reply:
That's definitely a more robust way of doing it. However, we'd still need to write out a solution time file for every restart file, so the name of the `ns-time.bin` file needs to be different to not overwrite the previous one.

Really, I should move over to using hdf5/cgns and store the timestep number and solution time in the restart file itself (unless there's an easy way of doing that with the binary files).

Reply:
We'd need a bit of code in PETSc to do parallel load from CGNS (I could give you pointers). The native HDF5 format does parallel IO, but the viz workflow is clunky.